docs/docker/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This script requires the local Docker image (playwright:localbuild) by the build.sh
+
+set -ex
+
+CONTAINER_ID="$(docker run --name playwright-docker-test -d -t playwright:localbuild /bin/bash)"
+
+# Reason for using a separate Git repo inside the Docker container instead of the current one is that 'docker cp'
+# Does not respect the .dockerignore or .gitignore
+docker cp $(pwd)/playwright-clean-state "${CONTAINER_ID}":/home/pwuser/playwright
+docker exec -u root "${CONTAINER_ID}" chown -R pwuser /home/pwuser/playwright/
+
+# Skip Electron tests
+docker exec --workdir /home/pwuser/playwright/ "${CONTAINER_ID}" rm -rf test/electron/
```

Comment:
why skip these?

Reply:
Due Chromium Sandbox, I can try to get them working.